app/Controllers/PrestitiController.php

```
@@ -1248,6 +1270,178 @@ public function downloadPdf(Request $request, Response $response, mysqli $db, in
+     */
+    public function bulkExtend(Request $request, Response $response, mysqli $db): Response
+    {
+        if ($guard = $this->guardStaffAccess($response)) {
+            return $guard;
+        }
+        $data = (array) $request->getParsedBody();
+        // CSRF validated by CsrfMiddleware.
+
+        $ids = array_values(array_unique(array_filter(
+            array_map('intval', (array) ($data['ids'] ?? [])),
+            static fn (int $i): bool => $i > 0
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $i. Configured minimum length is 3. (undefined)

(ShortVariable)